doc/builders/fetchers.chapter.md

```diff
@@ -68,3 +68,7 @@ This is used with Savannah repositories. The arguments expected are very similar
 ## `fetchFromRepoOrCz`
 
 This is used with repo.or.cz repositories. The arguments expected are very similar to fetchFromGitHub above.
+
+## `fetchFromSourcehut`
+
+This is used with sourcehut repositories. The arguments expected are very similar to fetchFromGitHub above.
```

Comment:
Probably should explain the quirk that the `owner` field needs to include the prefix.